probe-rs/src/architecture/riscv/communication_interface.rs

```diff
@@ -457,6 +457,11 @@ impl<'state> RiscvCommunicationInterface<'state> {
         // Reset error bits from previous connections
         self.dtm.clear_error_state()?;
 
+        // enable the debug module
```

Comment:
From what I recall, this move of the code is load bearing

Reply:
We can land these modifications outside of the P4 support anyway if we can give proper justification to them. Which I think we can, turning on the DM seems like a logical first step 😅